ikarus/finiteelements/mechanics/enhancedassumedstrains.hh

```diff
@@ -101,6 +101,7 @@ public:
    * \tparam RT The type representing the requested result.
    */
   template <template <typename, int, int> class RT>
+  requires(isSupportedResultType<SupportedResultTypes, RT>())
```

Comment:
Here, I didnt add it back then since also the displacement based FE can provide results that the EAS variant cant. Therefore, this now does not allow this anymore.

Reply:
Ahh okay i wondered why it didn't have that. So this also wouldn't work when using this Tuple declared here in EAS itself
https://github.com/ikarus-project/ikarus/blob/be7e0ae655c934a9dd3c7fa5146e0b348fa19839/ikarus/finiteelements/mechanics/enhancedassumedstrains.hh#L60